src/bcftools/bcftools_stats/config.vsh.yaml

```diff
@@ -24,191 +24,172 @@ argument_groups:
   - name: Inputs
     arguments:
       - name: --input
-        alternatives: -i
         type: file
         multiple: true
-        description: Input VCF/BCF file. Maximum of two files.
+        description: |
+          Input VCF/BCF file(s). Maximum of two files can be provided.
+          When two files are given, the program generates separate stats 
+          for intersection and the complements.
         required: true
+        example: input.vcf.gz
     
   - name: Outputs
     arguments:
       - name: --output
         alternatives: -o
         direction: output
         type: file
-        description: Output txt statistics file.
+        description: |
+          Write output to a file. The output is a text file which can be 
+          plotted using plot-vcfstats.
         required: true
+        example: stats.txt
          
   - name: Options
     arguments:
-      
-      - name: --allele_frequency_bins
-        alternatives: --af_bins
+      - name: --af_bins
         type: string
         description: | 
-          Allele frequency bins, a list of bin values (0.1,0.5,1).
-        example: 0.1,0.5,1
-
-      - name: --allele_frequency_bins_file
-        alternatives: --af_bins_file
-        type: file
-        description: | 
-          Same as allele_frequency_bins, but in a file.
-          Format of file is one value per line. 
-          e.g. 
-            0.1
-            0.5
-            1
+          Allele frequency bins, a list of comma-separated bin values.
+        example: "0.1,0.5,1"
 
-      - name: --allele_frequency_tag
-        alternatives: --af_tag
+      - name: --af_tag
         type: string
         description: | 
           Allele frequency tag to use, by default estimated from AN,AC or GT.
-
-      - name: --first_allele_only
-        alternatives: --first_only
-        type: boolean_true
-        description: | 
-          Include only 1st allele at multiallelic sites.
+        example: "AF"
 
       - name: --collapse
-        alternatives: --c
+        alternatives: -c
         type: string
-        choices: [ snps, indels, both, all, some, none ]
+        choices: [snps, indels, both, all, some, none]
```

Comment:
Above in the `--rm_dups` argument, the `none` option was removed. Should this be done here as well?

Reply:
No, this was an error in --rm_dups, but not in this argument.

bcftools concat:

```
   -d, --rm-dups STRING           Output duplicate records present in multiple files only once: <snps|indels|both|all|exact>
```

bcftools stats:

```
    -c, --collapse STRING            Treat as identical records with <snps|indels|both|all|some|none>, see man page for details [none]
```

Reply:
Ah, i see!